ddtrace/llmobs/_integrations/__init__.py

```diff
@@ -4,6 +4,10 @@
 from .gemini import GeminiIntegration
 from .langchain import LangChainIntegration
 from .openai import OpenAIIntegration
+from .utils import extract_model_name
```

Comment:
```suggestion
```

don't need to re-export these methods, we can just import from the `utils` module directly